### PR TITLE
Change a coalaTest.test_coala to use subprocess

### PR DESCRIPTION
**Pull request by Uran198**
An example of how https://github.com/coala/coala/issues/3403 might be done.

CC @Makman2

### CODE REVIEW

**Comment by sils** [comment]
how will this affect coverage measurement?

**Comment by jayvdb** [comment]
`coverage` can detect coverage through sub-processes. May needs some extra command line flags.

File: tests/coalaTest.py [hunk 1]
```diff
@@ -25,13 +27,16 @@ def tearDown(self):
     def test_coala(self):
         with bear_test_module(), \
```
Comment (sils) [inline tests/coalaTest.py]: you don't need the `with bear_test_module` on this one do you?

**Comment by Uran198** [comment]
This is just an example not intended to be merged in. I only wanted to run tests on it.

**Comment by Makman2** [comment]
@sils see https://github.com/coala/coala/issues/3403 if you want to discuss^^

**Pull request closed without merging**